GameTest.java

```diff
@@ -0,0 +1,215 @@
+		
+		// assert
+		assertEquals(Board.Creatures.Ghost1, this.game.board[l.getX()][l.getY() + 1]);
+		assertEquals(Board.Creatures.Null, this.game.board[l.getX()][l.getY()]);
+	}
+	
+	@Test
+	public void PacmanRightNull(){
+		// arrange
+		this.game.board = new Board(new String[]{"P."}).getBoard();
+		Location l = new Location(0, 0);
+		
+		// action
+		game.movePacman(l, Game.Directions.Right);
+		
+		// assert
+		assertEquals(Board.Creatures.Null, this.game.board[l.getX()] [l.getY()]);
+		assertEquals(Board.Creatures.Pacman , this.game.board[l.getX()] [l.getY() + 1]);
+	}
+
+	// Left
+	
+	@Test
+	public void PacmanLeftPoint(){
+		// arrange
+		this.game.board = new Board(new String[]{"-P"}).getBoard();
+		Location l = new Location(0, 1);	
+		
+		// action
+		game.movePacman(l, Game.Directions.Left);
+		
+		// assert
+		assertEquals(Board.Creatures.Null , this.game.board[l.getX()] [l.getY()]);
+		assertEquals(Board.Creatures.Pacman , this.game.board[l.getX()] [l.getY() - 1]);
+	}
+
+	// Up
+	
+	@Test
+	public void PacmanUpGhost(){
+		// arrange
+		this.game.board = new Board(new String[]{"G-", "PW"}).getBoard();
+		Location l = new Location(1, 0);
+		
+		// action
+		game.movePacman(l, Game.Directions.Up);
+		
+		// assert
+		assertEquals(Board.Creatures.Null , this.game.board[l.getX()] [l.getY()]);
+		assertEquals(Board.Creatures.Ghost1 , this.game.board[l.getX() - 1] [l.getY()]);
+	}
+
+	// Down
+	
+	@Test
+	public void PacmanDownWall(){
+		// arrange
+		this.game.board = new Board(new String[]{"PW", "WG"}).getBoard();
+		Location l = new Location(0, 0);
+		
+		// action
+		game.movePacman(l, Game.Directions.Down);
+		
+		// assert
+		assertEquals(Board.Creatures.Pacman , this.game.board[l.getX()] [l.getY()]);
+		assertEquals(Board.Creatures.Wall , this.game.board[l.getX() + 1] [l.getY()]);
+	}
+	
+	// - Ghosts -
+	// Right
+	
+	@Test
+	public void GhostRightWall(){
+		// arrange
+		this.game.board = new Board(new String[]{"GW"}).getBoard();
+		Location l = new Location(0, 0);
+		
+		// action
+		game.moveGhost1(l, Game.Directions.Right);
+		
+		// assert
+		assertEquals(Board.Creatures.Ghost1, this.game.board[l.getX()] [l.getY()]);
+		assertEquals(Board.Creatures.Wall, this.game.board[l.getX()] [l.getY() + 1]);
+	}
+
+	@Test
+	public void GhostRightPoint_FirstTurn(){
+		// arrange
+		this.game.board = new Board(new String[]{"G-"}).getBoard();
+		Location l = new Location(0, 0);
+		
+		// action
+		game.moveGhost1(l, Game.Directions.Right);
+		
+		// assert
+		assertEquals(Board.Creatures.Ghost1, this.game.board[l.getX()] [l.getY() + 1]);
+		assertEquals(Board.Creatures.Null, this.game.board[l.getX()] [l.getY()]);
+	}
+
+	@Test
+	public void GhostRightPoint_SecondTurn(){
+		// arrange
+		this.game.board = new Board(new String[]{"G-."}).getBoard();
+		Location l = new Location(0, 0);
+		
+		// action
+		game.moveGhost1(l, Game.Directions.Right);
+		game.moveGhost1(new Location(0, 1), Game.Directions.Right);
```

Comment:
It's better to use a coherent notation while using methods - 
`game.moveGhost1(l1, Game.Directions.Right);`
`game.moveGhost1(l2, Game.Directions.Right);`
or 
`game.moveGhost1(new Location(0, 0), Game.Directions.Right);`
`game.moveGhost1(new Location(0, 1), Game.Directions.Right);`

Try not to mix them